### PR TITLE
VIMC-4558: Update orderly web deploy to start redis & orderly workers

### PR DESCRIPTION
**Pull request by r-ash**
Note we need to remove the references to vimc-4192 when that has been merged into master

Order of PRs merging we need is
* orderly.server vimc-4192 https://github.com/vimc/orderly.server/pull/35
* orderlyweb mrc-2165 https://github.com/vimc/orderly-web/pull/295
* This PR
* A new orderlyweb R package PR - TODO

This builds on #28 too as travis was taking too long

### CODE REVIEW

File: config/complete/orderly-web.yml
Comment (richfitz) [inline config/complete/orderly-web.yml:84]: Do we want to hold off on merging this until vimc-4192 is merged on orderly.server? I guess you need this to fix the client

File: config/multiple-workers/orderly-web.yml
Comment (richfitz) [inline config/multiple-workers/orderly-web.yml:1]: you should be able to avoid this by modifying the above config at the point of tests (they accept a dictionary of overrides - there will be other examples in the tests). That will reduce keeping things in sync in future

**Review by richfitz — CHANGES_REQUESTED** [review]
One suggestion about simplifying the test config and a question about how we want to merge this

File: config/complete/orderly-web.yml
Comment (r-ash) [reply config/complete/orderly-web.yml:84]: Yep - I think the order will be (updating references at each stage) 
* orderly.server vimc-4192 vimc/orderly.server#35
* orderlyweb mrc-2165 vimc/orderly-web#295
* This PR
* A new orderlyweb R package PR - TODO
* Possibly also a change to python client

File: config/multiple-workers/orderly-web.yml
Comment (r-ash) [reply config/multiple-workers/orderly-web.yml]: Thanks - tidied that up

**Review by richfitz — APPROVED** [review]

**Pull request merged**
